lib/src/base-styles/_nx-page-layout.scss

```diff
@@ -8,6 +8,13 @@
 /*
   #Layout
 */
+:root {
```

Comment:
We already have a section for these at the bottom of `_nx-globals.scss`

Reply:
> We already have a section for these at the bottom of `_nx-globals.scss`

https://github.com/sonatype/sonatype-react-shared-components/pull/328/commits/9d6f28ce5dadad7cebf9d9e225d93582fed82f95